signbank/dictionary/adminviews.py

```diff
@@ -426,6 +435,9 @@ def validation_results_render_to_csv_response(self, context):
                 comment += f"{record.respondent_first_name} {record.respondent_last_name}: {record.comment} | "
             for share_comment in gloss_share_comment_map[gloss_record.pk]:
                 comment += f"{share_comment.user_name}: {share_comment.comment} | "
+            for manual_va in gloss_record.manual_vas:
+                if manual_va.comments:
+                    comment += f"{manual_va.group}: {manual_va.comments}"
```

Comment:
**question**: should this not be pipe-terminated like the others, in case there are multiple manual vas?

Reply:
yes 🤦🏻‍♀️